### DISCUSSION TITLE
Welcome to linkedin_login Discussions!

### DISCUSSION THREAD

**Discussion by d3xt3r2909** [d3xt3r2909/linkedin_login#43] [General]
## 👋 Welcome!

Questions, ideas, and most importantly good fun is welcome.

**Comment by sowsan** [comment]
How we can use this library with fire base authentication? Means after authorization create a firebase user. Thanks for your help.

**Reply by d3xt3r2909** [reply]
Hi, thanks for the question. To be honest I didn't implement this library in combination with the Firebase auth login so I think that's the best idea to ask this question on StackOverflow as you will have a much bigger community. As far as I know, the Firebase auth doesn't have LinkedIn as a supported provider for auth option, so maybe you can go in the direction to use firestore as a collection of users. But again, it's the only guesing.
